### PR TITLE
split data bugfix

### PR DESCRIPTION
**Pull request by lucidtronix**
yossi found a potentially serious&ugly bug in data splitting.  This should fix it also removed some dead code.

### CODE REVIEW

File: api_tutorials/chrom_hmm_cnn.py [hunk 1]
```diff
@@ -288,6 +288,8 @@ def bed_file_label(bed_dict, contig, pos):
 
 
 def split_data(datasets, valid_ratio=0.1, test_ratio=0.4):
+	assert(valid_ratio + test_ratio < 1.0)
```
Comment (yfarjoun) [inline api_tutorials/chrom_hmm_cnn.py]: while your at it, both should be >0

File: api_tutorials/chrom_hmm_cnn.py [hunk 2]
```diff
@@ -301,8 +303,8 @@ def split_data(datasets, valid_ratio=0.1, test_ratio=0.4):
 
 	for d in datasets:
 		valid.append( d[ :valid_idx] )
-		test.append(  d[valid_idx : test_idx] )
-		train.append( d[test_idx: ] )
+		test.append(  d[valid_idx : valid_idx+test_idx] )
```
Comment (yfarjoun) [inline api_tutorials/chrom_hmm_cnn.py]: spaces around `+` ?

**Comment by yfarjoun** [comment]
thanks.

File: api_tutorials/chrom_hmm_cnn.py (hunk 1 above)
Comment (lucidtronix) [reply api_tutorials/chrom_hmm_cnn.py]: done (I did >=, there could be times when you don't want the 3 way split)...

File: api_tutorials/chrom_hmm_cnn.py (hunk 2 above)
Comment (lucidtronix) [reply api_tutorials/chrom_hmm_cnn.py]: done

**Review by yfarjoun — APPROVED** [review]

**Pull request merged**